test/upgrade.js

```diff
@@ -14,7 +14,7 @@ describe("Contract Upgrades", function () {
   let signer2;
   let amountTransferredToSigner1;
 
-  before(async function () {
+  beforeEach(async function () {
```

Comment:
Why are we changing this back to `beforeEach` again? I thought we can keep `before` by changing the ownership back to `owner` (see suggested commit).

Reply:
I'd rather have a clean copy of both the base and upgraded contracts so each test is more self-contained, though it does work both ways.